lib/airport.rb

```diff
@@ -1,2 +1,33 @@
 class Airport
+  attr_accessor :planes
+  
+  MAX_CAPACITY = 12
+  
+  def initialize
+    @planes = []
+    3.times do
+      self.planes << Plane.new
+    end
+  end
+  
+  def release_plane(plane)
+    self.planes.delete plane
+  end
+  
+  def receive_plane(plane)
+    self.planes.push plane 
+  end
+  
+  def can_land?
+    current_weather? == :sunny && !self.full?
+  end
+  alias_method :can_take_off? :can_land?
```

Comment:
Hej Thomas!

Efter att ha suttit med Github (igår), sudo mm.(idag), så skulle jag börja
med att skriva min kod för airport, anropat rspec och när jag sedan skriver
in filnamnet i prompten så får jag som svar:
-bash: plane_spec.rb: command not found

Jag uppfattar det som att den inte hittar min fil och det kan jag inte
förstå.

/Erika

2015-09-06 18:21 GMT+02:00 Hound <notifications@github.com>:

> In lib/airport.rb
> <https://github.com/MakersSweden/airport_challenge/pull/1#discussion_r38823056>
> :
>
> > +      self.planes << Plane.new
> > +    end
> > +  end
> > +
> > +  def release_plane(plane)
> > +    self.planes.delete plane
> > +  end
> > +
> > +  def receive_plane(plane)
> > +    self.planes.push plane
> > +  end
> > +
> > +  def can_land?
> > +    current_weather? == :sunny && !self.full?
> > +  end
> > +  alias_method :can_take_off? :can_land?
>
> unexpected token tCOLON
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MakersSweden/airport_challenge/pull/1/files#r38823056>
> .
>